src/components/FormLabel/FormLabel.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './FormLabel.scss';
+
+const FormLabel = ({
```

Comment:
Why not `<Label />`?

Reply:
Not sure, figured we should make it specific, but I don't feel super strongly either way. Figured label might be too generic, but maybe it's not.